code/modules/mob/living/basic/vermin/mothroach.dm

```diff
@@ -0,0 +1,66 @@
+/mob/living/basic/mothroach
+	name = "mothroach"
+	desc = "An ancient ancestor of the moth that surprisingly looks like the crossbreed of a moth and a cockroach."
```

Comment:
Current moth lore does not permit precursor species to be present in the known world as the moth planet has been lost a long time ago, you could handwave this as being a product of genetic manipulation and it would be fine

Reply:
What do you think of "This is the adorable by-product of multiple attempts at genetically mixing mothpeople with cockroaches."?

Reply:
Sounds horrifying and probably unethical, but no lore objections!

Reply:
If you got any suggestions to make it sound more horrifying and unethical, I'm all for it!